easybuild/easyblocks/h/hdf5.py

```diff
@@ -92,13 +92,14 @@ def sanity_check_step(self):
         else:
             extra_binaries = ["bin/%s" % x for x in ["h5cc", "h5fc"]]
 
+        SHLIB_EXT = get_shared_lib_ext()
```

Comment:
'constants' (i.e., variables with a name in capitals) are only defined outside of classes/functions in the coding style we maintain, so please change this to:

```python
shlib_ext = get_shared_lib_ext()
```